src/Veldrid.Utilities/ObjParser.cs

```diff
@@ -372,33 +454,50 @@ public class ObjFile
 
         public ObjFile(Vector3[] positions, Vector3[] normals, Vector2[] texCoords, MeshGroup[] meshGroups, string materialLibName)
         {
-            Positions = positions;
-            Normals = normals;
-            TexCoords = texCoords;
-            MeshGroups = meshGroups;
+            Positions = positions ?? throw new ArgumentNullException(nameof(positions));
+            Normals = normals ?? throw new ArgumentNullException(nameof(normals));
+            TexCoords = texCoords ?? throw new ArgumentNullException(nameof(texCoords));
+            MeshGroups = meshGroups ?? throw new ArgumentNullException(nameof(meshGroups));
             MaterialLibName = materialLibName;
         }
 
         /// <summary>
         /// Gets a <see cref="ConstructedMeshInfo"/> for the given OBJ <see cref="MeshGroup"/>.
         /// </summary>
         /// <param name="group">The OBJ <see cref="MeshGroup"/> to construct.</param>
+        /// <param name="reduce">Whether to simplify the mesh by sharing identical vertices.</param>
         /// <returns>A new <see cref="ConstructedMeshInfo"/>.</returns>
-        public ConstructedMeshInfo GetMesh(MeshGroup group)
+        public ConstructedMeshInfo GetMesh(MeshGroup group, bool reduce = true)
```

Comment:
I tend to avoid default parameters in public API's. Can you change this to a required parameter and have the existing overload delegate to it w/ `true` ?

Reply:
Easy fix